modules/pallets/messaging-fees/src/impls.rs

```diff
@@ -225,27 +228,30 @@ where
 							*total = total.saturating_add(bytes_processed)
 						});
 						let current_total_bytes = TotalBytesProcessed::<T>::get();
-						let target_message_size = T::TargetMessageSize::get();
+						let target_message_size = Self::get_target_message_size();
 
 						if current_total_bytes < target_message_size {
 							let reward_amount = Self::calculate_reward(
 								current_total_bytes,
 								target_message_size,
 								base_reward,
 							)?;
-
-							T::Currency::transfer(
-								&T::TreasuryAccount::get().into_account_truncating(),
-								&relayer,
-								reward_amount.saturated_into(),
-								Preservation::Expendable,
-							)
-							.map_err(|_| Error::<T>::RewardTransferFailed)?;
-
-							Self::deposit_event(Event::FeeRewarded {
-								relayer: relayer.clone(),
-								amount: reward_amount.saturated_into(),
-							});
+							if reward_amount >= T::Currency::minimum_balance().into() {
+								T::Currency::transfer(
+									&T::TreasuryAccount::get().into_account_truncating(),
+									&relayer,
+									reward_amount.saturated_into(),
+									Preservation::Expendable,
+								)
+								.map_err(|_| Error::<T>::RewardTransferFailed)?;
+
+								Self::deposit_event(Event::FeeRewarded {
+									relayer: relayer.clone(),
+									amount: reward_amount.saturated_into(),
+								});
+							} else {
+								log::info!(target: "ismp", "Reward amount {reward_amount:?} below minimum balance");
+							}
```

Comment:
uhm maybe set the minimum balance to zero? otherwise people will miss rewards

Reply:
Or make the minimum balance absurdly small. eg 0.0000001 $BRIDGE

Reply:
Existential deposit has been changed, but keeping this just for the edge case where reward is zero